.github/workflows/maven.yml

```diff
@@ -0,0 +1,24 @@
+# This workflow will build a the Week 4 TDD Project and runs all Tests
+
+name: Java CI with Maven
+
+on:
+  push:
+    branches: [ dev-week4-tdd ]
+  pull_request:
+    branches: [ master, dev ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
```

Comment:
only ubuntu?

Reply:
This is the machine on which the tests run by GitHub.  Should I run on more platforms?

Reply:
No need, this should be good.